index.html

```diff
@@ -1,161 +1,129 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">  
-    <link href="https://fonts.googleapis.com/css2?family=Bai+Jamjuree:wght@400;600&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="build/css/style.css">
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="https://fonts.googleapis.com/css2?family=Bai+Jamjuree:wght@400;600&display=swap" rel="stylesheet" />
+    <link rel="stylesheet" href="build/css/style.css" />
     <title>Frontend Mentor | Clipboard landing page</title>
   </head>
   <body>
-    <main>
-
-      <header>
+    <main class="layout">
+      <header class="header">
         <div>
-          <h1>
-            A history of everything you copy
-          </h1>
-          <p>
-            Clipboard allows you to track and organize everything you 
-            copy. Instantly access your clipboard on all your devices.
-          </p>
-          <a href="#" class="btn btn-ios">
-            Download for iOS
-          </a>
-          <a href="#" class="btn btn-mac">
-            Download for Mac
-          </a>
+          <h1>A history of everything you copy</h1>
+          <p>Clipboard allows you to track and organize everything you copy. Instantly access your clipboard on all your devices.</p>
+          <a href="#" class="btn btn-ios"> Download for iOS </a>
+          <a href="#" class="btn btn-mac"> Download for Mac </a>
         </div>
       </header>
 
-      <section class="section1">
-        <div>
-          <h2>
-            Keep track of your snippets
-          </h2>
-          <p>
-            Clipboard instantly stores any item you copy in the cloud, 
-            meaning you can access your snippets immediately on all your 
-            devices. Our Mac and iOS apps will help you organize everything.
-          </p>
-        </div>
-        <div>
-          <ul>
-            <li>
-              <h3>
-                Quick Search
-              </h3>
-              <p>
-                Easily search your snippets by content, category, web address, application, and more.
-              </p>
+      <main class="content">
```

Comment:
there should be 1 main tag in every webpage

Reply:
I get it 😼